roles/tackle/templates/ingress-ui.yml.j2

```diff
@@ -3,7 +3,12 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
+{% if ui_ingress_class_name == 'nginx' %}
     nginx.ingress.kubernetes.io/proxy-body-size: {{ ui_ingress_proxy_body_size }}
+{% elif ui_ingress_class_name == 'alb' %}
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/scheme: internet-facing
+{% endif %}
```

Comment:
With two sets of annotations like this the default ansible k8s behavior when updating the value will be to add the new ones, without deleting the old.

I don't know if that will cause any interference, and I'd imagine in practice that won't happen often other than immediately after initial setup, realizing they can't access the UI, and changing it to the other.

Either way to prevent problems we can change the merge_type to merge on creation of this resource.

https://github.com/konveyor/tackle2-operator/blob/main/roles/tackle/tasks/main.yml#L426-L430

https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-merge_type

Reply:
thank you, I will come back to update the merge_type as you recommend